docs/rfc/block-registration.md

```diff
@@ -362,7 +362,7 @@ In the case of [dynamic blocks](/docs/designers-developers/developers/tutorials/
 
 ### `WPDefinedAsset`
 
-The `WPDefinedAsset` type is a subtype of string, where the value must represent an absolute or relative path to a JavaScript or CSS file.
+The `WPDefinedAsset` type is a subtype of string, where the value must represent a relative path to a JavaScript or CSS file.
```

Comment:
Do we really need an absolute path here if it is part of the plugin?

Reply:
> Do we really need an absolute path here if it is part of the plugin?

Probably not? I don't really recall the specific reason for this to have been a supported option.